.github/workflows/scheduled-istioCrypto-benchmark.yml

```diff
@@ -0,0 +1,153 @@
+        shell: bash
+
+  scheduled-test:
+    name: Scheduled Istio-Crypto Benchmark Test
+    timeout-minutes: 60
+    needs:
+      - start-runners-scheduled
+    runs-on: istioCrpto-fortio-load-${{ github.run_id }}
+    if: ${{ github.event_name == 'schedule' }}
+    strategy:
+      # fail-fast: false
+      # matrix:
+      #   service-mesh: ['istio', 'linkerd', 'osm']
+      #   load-generator: ['fortio', 'wrk2']
+      #   test-configuration: ['load-test','soak-test']
+    steps:
+      - name: Install dependencies
+        run: |
+          echo "Current user: $(whoami)"
+          echo "Installing kubectl..."
+          curl -LO https://dl.k8s.io/release/v1.23.2/bin/linux/amd64/kubectl 
+          sudo install -o smp -g smp -m 0755 kubectl /usr/local/bin/kubectl
+          echo "Installing docker..."
+          sudo apt update -y
+          sudo apt install -y jq unzip apt-transport-https ca-certificates software-properties-common
+          curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add -
+          sudo add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/ubuntu focal stable"
+          sudo apt-cache policy docker-ce
+          sudo apt install -y docker-ce
+          sudo systemctl status docker
+          sudo mkdir -p ~/.kube
+          sudo chmod 777 ~/.kube
+          curl -LO https://storage.googleapis.com/minikube/releases/latest/minikube-linux-amd64
+          sudo install minikube-linux-amd64 /usr/local/bin/minikube
+
+
+      - name: Start minikube 
+        run: |
+          minikube start --memory 5000 
+          kubectl get po -A
+
+      - name: Checkout Code
+        uses: actions/checkout@v2
+
+      - name: Deploy Istio Crypto design and deploy httpbin application
+        run: |
+          chmod +x .github/workflows/scripts/istioCrypto.sh
+          .github/workflows/scripts/istioCrypto.sh
+        shell: bash
+
+      - name: Get Date
+        id: date
+        run: echo "::set-output name=date::$(date +'%Y-%m-%d-%H.%M.%S')"
+        
+      - name: Run Benchmark Tests
+        uses: layer5io/meshery-smp-action@master
+        with:
+          provider_token: ${{ secrets.MESHERY_TOKEN }}
+          platform: docker
+          profile_filename: ${{ matrix.test-configuration }}.yaml
+          endpoint_url: ${{env.ENDPOINT_URL}}
+          service_mesh: ${{env.SERVICE_MESH}}
+          load_generator: ${{ matrix.load-generator }}
+          profile_name: 'istioCrypto-${{ matrix.test-configuration }}'
+          test_name: '${{ steps.date.outputs.date }}'
+
+  stop-runner-scheduled:
+    name: Stop self-hosted runner
+    timeout-minutes: 60
+    needs:
+      - start-runners-scheduled # required to get output from the start-runner job
+      - scheduled-test # required to wait when the main job is done
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        service-mesh: ['istio', 'linkerd', 'osm']
```

Comment:
It only works on Istio, so we don't need linkerd and osm.

Reply:
Yep I removed this